roles/percona-server/tasks/monitoring.yml

```diff
@@ -16,3 +16,8 @@
   notify: restart collectd
   when: collectd is defined and collectd.enabled|bool
   tags: collectd
+
+- name: disk check for /var/lib/mysql
+  sensu_check: name=disk-mysql plugin=check-disk-by-mount.rb args='-m /var/lib/mysql -w 70 -c 80'
```

Comment:
Given that we're already checking all paths except `/opt/stack/data` wouldn't this lead to double alerts should `/var/lib/mysql` fill up?